packages/slack-bot/src/utils.js

```diff
@@ -43,3 +45,25 @@ export function distancesFromEmbeddings({ queryEmbedding, embeddings }) {
   }))
   return distance
 }
+
+export async function downloadAudio(url, id) {
+  return new Promise(resolve => {
+    const u = new URL(url)
+    const dest = `./${id}.mp4`
+    fs.writeFile(dest, '')
```

Comment:
I think we don't need this. the file will be created when the response is piped to it